common/app/views/fragments/newHeader.scala.html

```diff
@@ -18,6 +18,18 @@
             <nav class="new-header__inner gs-container"
                  data-component="nav2"
                  aria-label="Guardian sections">
+
+                 @if(!page.metadata.hasSlimHeader) {
+                    <div class="new-header__edition-container hide-until-desktop">
+                         <a class="top-bar__item top-bar__item--dropdown"
+                             data-link-name=""
```

Comment:
should these have values?

Reply:
I've left them blank and Natalia is going to build in the functionality.